Makefile

```diff
@@ -51,14 +51,35 @@ pre-build:
 	@ar rvs ${OUTDIR}/libindexer.a ${OUTDIR}/indexer.o
 	@rm -f ${OUTDIR}/indexer.o
 
+install-cmake:
+	@wget https://github.com/Kitware/CMake/releases/download/v3.24.1/cmake-3.24.1-Linux-x86_64.sh -O cmake.sh
+	@sh cmake.sh --prefix=/usr/local/ --exclude-subdir
+	@rm -rf cmake.sh
+
+install-flatc:
```

Comment:
How about using compiled binaries from https://github.com/google/flatbuffers/releases instead? Installing cmake for a Go project seems too much to me.